pandas/core/series.py

```diff
@@ -2855,6 +2860,9 @@ def _try_kind_sort(arr):
 
         result = self._constructor(arr[sortedIdx], index=self.index[sortedIdx])
 
+        if ignore_index:
```

Comment:
can you rename sortedIdx -> sorted_index  in this method

Reply:
renamed